website/client/components/chat/chatMessages.vue

```diff
@@ -207,12 +189,6 @@ export default {
       this.lastOffset = container.scrollTop - (container.scrollHeight - container.clientHeight);
       // disable scroll
       container.style.overflowY = 'hidden';
-
-      const canLoadMore = this.inbox && !this.isLoading && this.canLoadMore;
```

Comment:
@SabreCat was this removed by intention ? or just too much refactored ^^

Reply:
I see the inbox-part `chatMessages` was extracted to `messageList`, currently I can't load more messages, anyway now I know where to look correctly ^^